logging/api_reference/_attributes

```diff
@@ -0,0 +1 @@
+../_attributes/
```

Comment:
General comment for the PR. While going up one level with `../` will work, as it will point to the sym link in `logging`, I am pretty sure you want to go up two levels. So in this case `../../_attributes/`.